packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -398,16 +400,21 @@ export async function activate(context: vscode.ExtensionContext) {
     sfdxProjectOpened
   );
 
+  let defaultUsernameorAlias: string | undefined;
+  if (hasRootWorkspace()) {
+    defaultUsernameorAlias = await OrgAuthInfo.getDefaultUsernameOrAlias(false);
+  }
+
   // register org picker commands and set up filewatcher for defaultusername
   const orgList = new OrgList();
-  await orgList.displayDefaultUsername();
+  await orgList.displayDefaultUsername(defaultUsernameorAlias);
```

Comment:
You can remove this `await` after removing the `async` from `displayDefaultUsername`